cms/src/components/content/ContentOverview.spec.ts

```diff
@@ -0,0 +1,176 @@
+import { describe, it, expect, vi, beforeAll, afterAll } from "vitest";
+import { mount } from "@vue/test-utils";
+import { createTestingPinia } from "@pinia/testing";
+import ContentOverview from "@/components/content/ContentOverview.vue";
+import {
+    fullAccessToAllContentMap,
+    mockEnglishContentDto,
+    mockFrenchContentDto,
+    mockLanguageDtoEng,
+    mockPostDto,
+    viewAccessToAllContentMap,
+} from "@/tests/mockData";
+import { setActivePinia } from "pinia";
+import { useUserAccessStore } from "@/stores/userAccess";
+import { DocType } from "@/types";
+import { ref } from "vue";
+import { DateTime } from "luxon";
+import { RouterLink, type RouteLocationNamedRaw } from "vue-router";
+import { EyeIcon, PencilSquareIcon } from "@heroicons/vue/20/solid";
+
+describe("ContentOverview.vue", () => {
+    beforeAll(async () => {
+        vi.mock("@/db/baseDatabase", () => ({
+            db: {
+                whereTypeAsRef: vi.fn((docType) => {
+                    if (docType === "post") {
+                        return ref([mockPostDto]);
+                    } else if (docType === "language") {
+                        return ref([mockLanguageDtoEng]);
+                    }
+
+                    return ref([]);
+                }),
+                whereParentAsRef: vi.fn(() => {
+                    return ref([mockEnglishContentDto]);
+                }),
+                isLocalChange: vi.fn(() => {
+                    return false;
+                }),
+                toDateTime: vi.fn((val) => {
+                    return DateTime.fromMillis(val);
+                }),
+            },
+        }));
```

Comment:
Shall we remove the database mocks and rather test using mock date added to indexeddb? (Same as in EditContent.spec.ts)